ffi/native/dart_vlc.cpp

```diff
@@ -13,7 +13,7 @@
 #ifdef _WIN32
 #define DLLEXPORT __declspec(dllexport)
 #else
-#define DLLEXPORT
+#define DLLEXPORT __attribute__((used))
```

Comment:
Maybe we might need another `#elif __linux__` here or any Apple specific macro.

Reply:
It's a attribute supported by gcc-like compilers, eg. [gcc](https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html#:~:text=for%20this%20function.-,used,-This%20attribute%2C%20attached) and [clang](https://clang.llvm.org/docs/AttributeReference.html#used). We can use `#elif __GNUC__` if other compilers are considerated.

Reply:
`__attribute__((used))` is needed here because the dartvlc wrapper is statically linked on macOS currently, and these funtions are not referenced by any other libraries or the executable so they can be removed(or stripped) by the linker by default, even some global initialization with side-effect exists.

Reply:
@namniav okay.